### DISCUSSION TITLE
Should Microdiff on NPM be minified?

### DISCUSSION THREAD

**Discussion by AsyncBanana** [AsyncBanana/microdiff#8] [Ideas]
In #5, @simone-sanfratello brought up minifying the code in Microdiff's NPM package. However, most people targeting the web already minify everything themselves, and many people who use Node like their dependencies being readable.
This discussion is for seeing what the community's opinions on this are. If you have an opinion on this, please react with :+1: or :-1: and add any other thoughts in the replies.

**Comment by vegeta897** [comment]
Just to be clear, thumbs down = should not be minified?

Any package installed from NPM should absolutely not be minified. Anyone who is interested in delivering a small app is already using a minifier for the rest of their code. Being able to read a dependency's code (preferably with comments intact) is very valuable, and it also gives cleaner error traces. Minified versions of the package can be _included_ in the NPM distribution for those that want them, but it should not be what is used as the entry point for imports.

**Reply by AsyncBanana** [reply]
Yeah, 👎 is saying it should not be minified, as Microdiff currently is

**Comment by AsyncBanana** [comment]
Due to the majority that says it should not be minified, I will keep it as it is currently.
